CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.12.0] - released on 2017-05-31
+### Added
+- Added `generate_source_artifacts` method to the platform dsl to allow generation
+  of `.src.rpm` and debian source artifacts.
+- Added the ability to specify mirror URLs.
+
+### Deprecated
+- The existing rewrite engine is being deprecated in favor of the mirror URLs.
+
+### Changed
+- Updated to sending notification messages on STDERR instead of STDOUT for easier filtering.
```

Comment:
This is also pretty meaningful so I'd probably add a couple of lines about what/why and reference the PR where it went in for context?